proto/public.proto

```diff
@@ -366,7 +366,8 @@ message TopRoutesRequest {
 
   oneof outbound {
     Empty none = 3;
-    Resource from_resource = 5;
+    string to_service = 5;
+    Empty to_all = 6;
```

Comment:
Given the CLI flag changes I suggested above, and for consistency with the `StatSummaryRequest` struct, I'd change this to:

```
  oneof outbound {
    Empty none = 3;
    Resource to_resource = 4;
  }
```